openedx/core/djangoapps/util/signals.py

```diff
@@ -19,7 +19,7 @@ def record_request_exception(sender, **kwargs):
     Logs the stack trace whenever an exception
     occurs in processing a request.
     """
-    logging.exception(u"Uncaught exception from {sender}".format(
+    logging.exception("Uncaught exception from {sender}".format(
         sender=sender
     ))
```

Comment:
```suggestion
    logging.exception(f"Uncaught exception from {sender}")
```

Reply:
Using `format` is preffered in case of logging as compared to `f-string` method.